### PR TITLE
Update group_id syntax

### LINKED ISSUE #62: [bug] group_id is not recognised anymore
Using latest `fink_client` version:

```
registering new views: args: ('groups', <class 'tom_common.api_views.GroupViewSet'>, 'groups'), kwargs: {}
registering new views: args: ('targets', <class 'tom_targets.api_views.TargetViewSet'>, 'targets'), kwargs: {}
registering new views: args: ('targetextra', <class 'tom_targets.api_views.TargetExtraViewSet'>, 'targetextra'), kwargs: {}
registering new views: args: ('targetname', <class 'tom_targets.api_views.TargetNameViewSet'>, 'targetname'), kwargs: {}
registering new views: args: ('targetlist', <class 'tom_targets.api_views.TargetListViewSet'>, 'targetlist'), kwargs: {}
registering new views: args: ('observations', <class 'tom_observations.api_views.ObservationRecordViewSet'>, 'observations'), kwargs: {}
registering new views: args: ('dataproducts', <class 'tom_dataproducts.api_views.DataProductViewSet'>, 'dataproducts'), kwargs: {}
registering new views: args: ('reduceddatums', <class 'tom_dataproducts.api_views.ReducedDatumViewSet'>, 'reduceddatums'), kwargs: {}
NumExpr defaulting to 8 threads.
Exception in thread FinkAlertStream:
read_streams FinkAlertStream TID=475562 ; thread identifier=125194441811520
Traceback (most recent call last):
  File "/home/peloton/anaconda3/lib/python3.9/threading.py", line 973, in _bootstrap_inner
readstreams Command.handle() returning...
    self.run()
  File "/home/peloton/anaconda3/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
  File "/home/peloton/anaconda3/lib/python3.9/site-packages/tom_fink/alertstream.py", line 85, in listen
    consumer = AlertConsumer([self.topic], myconfig, schema_path=None)
  File "/home/peloton/codes/fink-client/fink_client/consumer.py", line 76, in __init__
    self._kafka_config = _get_kafka_config(config)
  File "/home/peloton/codes/fink-client/fink_client/consumer.py", line 524, in _get_kafka_config
    kafka_config["group.id"] = config["group.id"]
KeyError: 'group.id'
```

### PR DESCRIPTION
**Pull request by JulienPeloton**
Closes #62 

This PR updates the syntax for the `group.id` parameter for Kafka in fink-client. I also updated the required version of fink-client & minimum version of Python.

### CODE REVIEW

File: tom_fink/alertstream.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def listen(self):
         while poll_number < int(self.max_poll_number):
             try:
                 logger.info(
-                    "FinkAlertStream.listen opening stream: {} with group_id: {} (call number: {})".format(
+                    "FinkAlertStream.listen opening stream: {} with group.id: {} (call number: {})".format(
```
Comment (phycodurus) [inline tom_fink/alertstream.py:91]: Hi Julien, thanks for keeping tom_fink up-to-date. How about using an f-string here?
```
logger.info(f"FinkAlertStream.listen opening stream {self.url} with group.id {self.group_id} (call number: {poll_number})")
```

or this also works if the line gets too long:

```
logger.info(f"FinkAlertStream.listen opening stream {self.url}"
            f" with group.id {self.group_id} (call number: {poll_number})")
```

If you like that syntax, there are other opportunities to convert from `"".format()` to f-string syntax around lines 138-143 and 156-158.

Thanks again!

**Review by phycodurus — APPROVED** [review]

File: tom_fink/alertstream.py (hunk 1 above)
Comment (JulienPeloton) [reply tom_fink/alertstream.py]: Thanks @phycodurus  for the review -- I converted to f-string when it was simple enough -- but I prefer keeping the `.format` syntax when it involves dictionaries (clearer to my opinion).

**Comment by JulienPeloton** [comment]
@phycodurus I'll let you merge (I'm no more authorized to do it?)

**Comment by jchate6** [comment]
> @phycodurus I'll let you merge (I'm no more authorized to do it?)

Hi @JulienPeloton!
Sorry this was my fault. I'm going through adding some checks and security to our TOM Repositories. 
I was intending to just make it have a check requiring approval by a reviewer before someone could push a PR to main.
I think I've fixed this now, and you should be able to push. 

If you are still having issues, let me know and I'll put it back how it was before.

**Comment by JulienPeloton** [comment]
Thank you @jchate6 -- the current setup is perfect!

**Pull request merged**